lib/safely_block.rb

```diff
@@ -12,13 +12,11 @@ def report_exception(e)
   end
 
   DEFAULT_EXCEPTION_METHOD = proc do |e|
-    e = e.dup # leave original exception unmodified
-    e.message.prepend("[safely] ") if e.message && Safely.tag
     Errbase.report(e)
   end
 
   self.env = ENV["RACK_ENV"] || ENV["RAILS_ENV"] || "development"
-  self.tag = true
+  self.tag = "safely"
```

Comment:
For backward compatibility, we'll need to support the current behavior with `Safety.tag = true`.